pkg/values/secret.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // MergeSecretData merges the data from the catalog, app and user secretss
 // and returns a single set of values.
-func (v *Values) MergeSecretData(ctx context.Context, app v1alpha1.App, appCatalog v1alpha1.AppCatalog) (map[string][]byte, error) {
+func (v *Values) MergeSecretData(ctx context.Context, app v1alpha1.App, appCatalog v1alpha1.AppCatalog) (map[string]interface{}, error) {
```

Comment:
There is no reason to return map[string][]byte type in here, it could be done by each operator.

Reply:
The types were because configmaps use map[string]string and secrets use map[string][]byte.

I'm fine to change it but it breaks the interface so this needs to be a major bump. WDYT?

Reply:
Sure, even though it does not belong to the interface, I think it's better to bump up. Thanks for noticing.